point-focus.js

```diff
@@ -1,7 +1,7 @@
 document.addEventListener("DOMContentLoaded", function () {
     document.body.addEventListener("mousedown", function (evt) {
-        if (!evt.target.setSelectionRange || evt.target.role === 'textbox' || evt.target.hasAttribute("disable-point-focus")) {
-                evt.target.setAttribute("point-focused", true); 
+        if (!t.setSelectionRange && t.getAttribute("role") !== 'textbox' || t.hasAttribute("disable-point-focus")) {
+            evt.target.setAttribute("point-focused", true); 
```

Comment:
Also, this is setting point-focused on the event target, not `document.body`, but the updated test page assumes it's on `body`?